packages/adaptive_scaffold/lib/src/adaptive_layout.dart

```diff
@@ -0,0 +1,517 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+import 'package:flutter/widgets.dart';
+import 'breakpoints.dart';
+import 'slot_layout.dart';
+
+enum _SlotIds {
+  primaryNavigation,
+  secondaryNavigation,
+  topNavigation,
+  bottomNavigation,
+  body,
+  secondaryBody,
+}
+
+/// [AdaptiveLayout] is dedicated to simplifying the process of developing
+/// layouts for many different screen sizes. It separates the screen into
+/// several sections, also referred to as "slots". The Widget defines slots for
+/// the different placements of navigational elements and two slots for a body
+/// and secondaryBody. All of these slots take a [SlotLayout] Widget. This class
+/// works in junction with [SlotLayout] and [SlotLayoutConfig]. This class
```

Comment:
These two sentence don't quite make sense.  I think the overall goal here is to explain what classes are used to defined what each slot means.